tabular/src/autogluon/tabular/models/automm/automm_model.py

```diff
@@ -215,3 +215,10 @@ def get_minimum_resources(self) -> Dict[str, int]:
     def _more_tags(self):
         # `can_refit_full=False` because MultiModalPredictor does not communicate how to train until the best epoch in refit_full.
         return {'can_refit_full': False}
+
+    @classmethod
+    def _get_default_ag_args_ensemble(cls, **kwargs) -> dict:
+        default_ag_args_ensemble = super()._get_default_ag_args_ensemble(**kwargs)
+        extra_ag_args_ensemble = {'fold_fitting_strategy': 'sequential_local'}
+        default_ag_args_ensemble.update(extra_ag_args_ensemble)
+        return default_ag_args_ensemble
```

Comment:
This is now part of mainline, feel free to remove and rebase: https://github.com/awslabs/autogluon/pull/2308